src/funnels/Depositor.sol

```diff
@@ -118,15 +120,12 @@ contract Depositor {
         emit Deny(usr);
     }
 
-    function setLimits(address gem0, address gem1, uint64 hop, uint128 cap0, uint128 cap1) external auth {
+    function setLimits(address gem0, address gem1, uint128 cap0, uint128 cap1, uint64 hop) external auth {
         require(gem0 < gem1, "Depositor/wrong-gem-order");
-        limits[gem0][gem1] = PairLimit({
-            hop:  hop,
-            zzz:  limits[gem0][gem1].zzz,
-            cap0: cap0,
-            cap1: cap1
-        });
-        emit SetLimits(gem0, gem1, hop, cap0, cap1);
+        limits[gem0][gem1].cap0 = cap0;
+        limits[gem0][gem1].cap1 = cap1;
+        limits[gem0][gem1].hop = hop;
```

Comment:
I'm wondering if she should maybe reset the `limits[gem0][gem1].amt{i}` here when `cap{i}` is changed so that a reduction of `cap{i}` cannot temporarily be ignored in case  the existing `amt{i} > newCap{i}`. 

Or alternatively I guess we could just leave a note warning that a change of cap will only become effective after (up to) `hop` amount of time.

Reply:
Changed it in both Depositor and Swapper, we might evaluate to directly set the amounts to 0 or just make both values as arguments and let the subdao proxy to define the values depending on the needs of the case.

Reply:
I think current code in the Depositor could bring a weird state where one amt is updated to the cap and one is not, maybe just simpler to indeed always reset to cap (also in the Swapper for alignment).

Reply:
when you say 0 you mean set both to the new `cap`s right?

Reply:
Yeah I guess both are valid options, we can set to 0 and wait for the new `hop` to pass (if didn't) or set to `cap`, which might allow two operations for the full `cap` pretty soon if the time to surpass the new `hop` is close to happen.
That's was the main reason why I felt keeping the amount or setting to new `cap` is higher was the most fair situation.

Reply:
To keep things simple, I'd be tempted to reset the state in a way that looks similar to the initial state: set the `due` amounts to 0 and set `zzz` to 0 as well. 

That means that will be possible to spend `existingDue` + `newCap` back to back but that seems like an acceptable trade-off occurring at the boundaries between new configs.

Reply:
Done.